src/applications/mhv-medical-records/tests/fixtures/labsAndTests3.json

```diff
@@ -0,0 +1,279 @@
+          "category": [
+              {
+                  "coding": [
+                      {
+                          "system": "http://terminology.hl7.org/CodeSystem/observation-category",
+                          "code": "laboratory"
+                      }
+                  ]
+              }
+          ],
+          "code": {
+              "text": "Parasitology Remark(s)"
+          },
+          "subject": {
+              "reference": "Patient/ex-MHV-patient-1"
+          },
+          "effectiveDateTime": "1995-08-01T11:07:00Z",
+          "performer": [
+              {
+                  "reference": "#ex-MHV-organization-989"
+              }
+          ],
+          "valueString": "MODERATE YEAST"
+      },
+      {
+          "resourceType": "Specimen",
+          "id": "ex-MHV-specimen-3",
+          "meta": {
+              "profile": [
+                  "https://department-of-veterans-affairs.github.io/mhv-fhir-phr-mapping/StructureDefinition/VA.MHV.PHR.LabSpecimen"
+              ]
+          },
+          "identifier": [
+              {
+                  "use": "usual",
+                  "system": "urn:oid:2.16.840.1.113883.4.349.4.989",
+                  "value": "LabSpecimenTO.6Y100"
+              }
+          ],
+          "accessionIdentifier": {
+              "use": "usual",
+              "system": "urn:oid:2.16.840.1.113883.4.349.4.989",
+              "value": "PARAS 95 262"
+          },
+          "status": "available",
+          "type": {
+              "text": "STOOL"
+          },
+          "collection": {
+              "collectedDateTime": "1995-07-28",
+              "bodySite": {
+                  "text": "FECES"
+              }
+          }
+      },
+      {
+          "resourceType": "Organization",
+          "id": "ex-MHV-organization-989",
+          "meta": {
+              "profile": [
+                  "https://department-of-veterans-affairs.github.io/mhv-fhir-phr-mapping/StructureDefinition/VA.MHV.PHR.organization"
+              ]
+          },
+          "identifier": [
+              {
+                  "use": "usual",
+                  "system": "urn:oid:2.16.840.1.113883.4.349",
+                  "value": "LabSiteTO.989"
+              },
+              {
+                  "system": "http://hl7.org/fhir/sid/us-npi",
+                  "value": "1234"
+              }
+          ],
+          "active": true,
+          "name": "Lab Site 989"
+      },
+      {
+          "resourceType": "Practitioner",
+          "id": "ex-MHV-practitioner-mangas",
+          "meta": {
+              "profile": [
+                  "http://hl7.org/fhir/us/core/StructureDefinition/us-core-practitioner"
+              ]
+          },
+          "identifier": [
+              {
+                  "system": "http://nowhere.com/nope",
+                  "value": "unknown"
+              }
+          ],
+          "name": [
+              {
+                  "text": "MANGAS,PHYLLIS A",
```

Comment:
For any names that look real in your fixtures, can you modify them to be more generic? e.g. Jane Doe?